lib/pdn/Kconfig

```diff
@@ -7,11 +7,15 @@
 menuconfig PDN
 	bool "Packet Domain Network library"
 	depends on NRF_MODEM_LIB && AT_MONITOR
-	# library uses heap
-	depends on HEAP_MEM_POOL_SIZE > 0
 
 if PDN
 
+config HEAP_MEM_POOL_ADD_SIZE_PDN
```

Comment:
Oh, nice-- I didn't know it worked like that. Definitly more correct to do it this way.